DirectEmployers/MyJobs
# [PD-4029] Fix parsing issue with location search.

## What's this PR do?
Fixes an issue were location searches were getting over-escaped and ended up just being wrong in general. For an example of this, see the description in PD-4029.

## Where should the reviewer start?
Small fix in helpers.py

## How should this be manually tested?
Load a bunch of jobs in for a single state (Indiana)
Search for one location in that state (Indianapolis, IN)
Before the fix, you get all the jobs in Indiana when searching Indianapolis, IN. After the fix, you should only get jobs in Indianapolis.

## Any background context you want to provide?
Haystack is annoying sometimes.

## PR Checklist

### Separation of Concerns
